### ISSUE TITLE
NPE in dependency_computer.dart

### ISSUE THREAD

**Issue by Andersmholmgren** [dart-lang/pub#1286]
running pub build within docker (within gcloud)

```
INFO     2015-06-24 00:13:23,267 containers.py:280] 
INFO     2015-06-24 00:13:27,609 containers.py:280] Changed 2 dependencies!
INFO     2015-06-24 00:13:27,701 containers.py:280] Precompiling dependencies...
INFO     2015-06-24 00:13:27,732 containers.py:280] Loading source assets...
INFO     2015-06-24 00:13:28,155 containers.py:280] The null object does not have a getter 'pubspec'.

NoSuchMethodError: method not found: 'pubspec'
Receiver: null
Arguments: []

INFO     2015-06-24 00:13:28,178 containers.py:280] dart:core                                                                                            Object.noSuchMethod
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 251   _PackageDependencyComputer._PackageDependencyComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 107   DependencyComputer._transformersNeededByTransformer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 258   _PackageDependencyComputer._PackageDependencyComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
dart:core                                                                                            List.forEach
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 53    DependencyComputer.DependencyComputer
INFO     2015-06-24 00:13:28,179 containers.py:280] 
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart 33  loadAllTransformers.<async>
dart:isolate                                                                                         _RawReceivePortImpl._handleMessage

INFO     2015-06-24 00:13:28,975 containers.py:280] ---> 8c0f30e0002c
INFO     2015-06-24 00:13:29,039 containers.py:280] Removing intermediate container e3f214a9c41b
INFO     2015-06-24 00:13:29,040 containers.py:280] Step 12 : RUN pub --trace build
INFO     2015-06-24 00:13:29,128 containers.py:280] ---> Running in 890b1bc370da
INFO     2015-06-24 00:13:30,177 containers.py:280] Loading source assets...
INFO     2015-06-24 00:13:30,178 containers.py:280] 
INFO     2015-06-24 00:13:31,082 containers.py:280] The null object does not have a getter 'pubspec'.

NoSuchMethodError: method not found: 'pubspec'
Receiver: null
Arguments: []

INFO     2015-06-24 00:13:31,145 containers.py:280] dart:core                                                                                            Object.noSuchMethod
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 251   _PackageDependencyComputer._PackageDependencyComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 107   DependencyComputer._transformersNeededByTransformer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 258   _PackageDependencyComputer._PackageDependencyComputer
INFO     2015-06-24 00:13:31,146 containers.py:280] 
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
dart:core                                                                                            List.forEach
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 53    DependencyComputer.DependencyComputer
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart 33  loadAllTransformers.<async>
dart:isolate                                                                                         _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                            _Completer.completeError
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart 127  loadAllTransformers.<async>
dart:isolate                                                                                          _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                        Future.Future.microtask
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart  loadAllTransformers
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 512  AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>.<async>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart      AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                        progress
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 511  AssetEnvironment._load.<fn>.<async>.<fn>.<async>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                        Future.Future.sync
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 671  AssetEnvironment._withStreamErrors
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 510  AssetEnvironment._load.<fn>.<async>
dart:async                                                                                        _SyncCompleter.complete
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 672  AssetEnvironment._withStreamErrors.<fn>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                        _Future.then
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 488  AssetEnvironment._load.<fn>.<async>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
INFO     2015-06-24 00:13:31,148 containers.py:280] 
===== asynchronous gap ===========================
dart:async                                                                                        _Future.then
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 477  AssetEnvironment._load.<fn>.<async>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                        Future.Future.microtask
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart      AssetEnvironment._load.<fn>
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                        progress
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 459  AssetEnvironment._load
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 82   AssetEnvironment.create.<fn>.<async>
dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                       _Future.then
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 74  AssetEnvironment.create.<fn>.<async>
dart:isolate                                                                                     _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                                       Future.Future.microtask
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart     AssetEnvironment.create.<fn>
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                       progress
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 73  AssetEnvironment.create
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/build.dart 56              BuildCommand.onRunTransformerCommand.<async>
dart:isolate                                                                                     _RawReceivePortImpl._handleMessage
===== asynchronous gap ===========================
dart:async                                                                             Future.Future.microtask
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/build.dart       BuildCommand.onRunTransformerCommand
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/barback.dart 58  BarbackCommand.run
package:args/command_runner.dart 179                                                   CommandRunner.runCommand.<fn>
dart:async                                                                             Future.Future.sync
package:args/command_runner.dart 132                                                   CommandRunner.runCommand
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command_runner.dart 122  PubCommandRunner.runCommand.<async>.<fn>
dart:async                                                                             Future.Future.sync
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/utils.dart 107           captureErrors.<fn>
package:stack_trace/src/chain.dart 78                                                  Chain.capture
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/utils.dart 121           captureErrors
/tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command_runner.dart 122  PubCommandRunner.runCommand.<async>
dart:isolate                                                                           _RawReceivePortImpl._handleMessage
---- Log transcript ----
FINE: Pub 1.10.0
FINE: Loading asset environment...
FINE: Loading package graph...
IO  : Spawning "git --version" in /app/gitbacklog_client/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | git version 1.7.10.4
    | Nothing output on stderr.
FINE: Determined git command null.
IO  : Spawning "git rev-list --max-count=1 c64ab8d2067d08a05b3e236cf791dcd2ebbf555e" in /root/.pub-cache/git/cache/frentity-4e42bd2b63e1a65801e159a606e4d635f4695777
IO  : Spawning "git rev-list --max-count=1 3cdc017b6306044c3dea41538a3d40f2a69bb2e7" in /root/.pub-cache/git/cache/gissue_common-91a57708d23c42c88b790226dce46417a7d3e8ea
IO  : Spawning "git rev-list --max-count=1 abbb2980eba39693d2cfc2b9df3fb7e0c371c528" in /root/.pub-cache/git/cache/ah_polymer_stuff-d7795d1d2bf5bf41e1dec771dfd2e433c44e624c
IO  : Spawning "git rev-list --max-count=1 5e31c2f8c9dae8513c9003480e7aa755683e7297" in /root/.pub-cache/git/cache/gissue_client-9fb324d5599bf96a55872223e8c475f09efe6166
IO  : Spawning "git rev-list --max-count=1 c57870f7c92d7aa4c57141ff4a7232e24be8ceec" in /root/.pub-cache/git/cache/polymer_expressions-46d6a74de12add1443b44588a2c9e8e8a97699ba
IO  : Finished git. Exit code 0.
    | stdout:
    | | c64ab8d2067d08a05b3e236cf791dcd2ebbf555e
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | 3cdc017b6306044c3dea41538a3d40f2a69bb2e7
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | abbb2980eba39693d2cfc2b9df3fb7e0c371c528
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | 5e31c2f8c9dae8513c9003480e7aa755683e7297
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | c57870f7c92d7aa4c57141ff4a7232e24be8ceec
    | Nothing output on stderr.
IO  : Spawning "git rev-list --max-count=1 c64ab8d2067d08a05b3e236cf791dcd2ebbf555e" in /root/.pub-cache/git/cache/frentity-4e42bd2b63e1a65801e159a606e4d635f4695777
IO  : Spawning "git rev-list --max-count=1 3cdc017b6306044c3dea41538a3d40f2a69bb2e7" in /root/.pub-cache/git/cache/gissue_common-91a57708d23c42c88b790226dce46417a7d3e8ea
IO  : Spawning "git rev-list --max-count=1 abbb2980eba39693d2cfc2b9df3fb7e0c371c528" in /root/.pub-cache/git/cache/ah_polymer_stuff-d7795d1d2bf5bf41e1dec771dfd2e433c44e624c
IO  : Spawning "git rev-list --max-count=1 5e31c2f8c9dae8513c9003480e7aa755683e7297" in /root/.pub-cache/git/cache/gissue_client-9fb324d5599bf96a55872223e8c475f09efe6166
IO  : Spawning "git rev-list --max-count=1 c57870f7c92d7aa4c57141ff4a7232e24be8ceec" in /root/.pub-cache/git/cache/polymer_expressions-46d6a74de12add1443b44588a2c9e8e8a97699ba
IO  : Finished git. Exit code 0.
    | stdout:
    | | c64ab8d2067d08a05b3e236cf791dcd2ebbf555e
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:

INFO     2015-06-24 00:13:31,149 containers.py:280]     | | 3cdc017b6306044c3dea41538a3d40f2a69bb2e7
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | abbb2980eba39693d2cfc2b9df3fb7e0c371c528
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | 5e31c2f8c9dae8513c9003480e7aa755683e7297
    | Nothing output on stderr.
IO  : Finished git. Exit code 0.
    | stdout:
    | | c57870f7c92d7aa4c57141ff4a7232e24be8ceec
    | Nothing output on stderr.
FINE: Loading package graph finished (0.4s).
FINE: Initializing barback...
FINE: Serving packages on localhost:0.
MSG : Loading source assets...
FINE: Providing sources for web_components|lib.
FINE: Providing sources for smoke|lib.
FINE: Providing sources for polymer_interop|lib.
FINE: Providing sources for bwu_fontawesome_iconset_svg|lib.
FINE: Providing sources for observe|lib.
FINE: Providing sources for dart_style|lib.
FINE: Providing sources for core_elements|lib.
FINE: Providing sources for initialize|lib.
FINE: Providing sources for polymer|lib.
FINE: Providing sources for paper_elements|lib.
FINE: Providing sources for ah_polymer_stuff|lib.
FINE: Providing sources for gissue_client|lib.
FINE: Providing sources for gitbacklog_client|lib.
FINE: Loading source assets finished (0.6s).
FINE: Provided sources.
FINE: Loading transformers...
FINE: Loading transformers finished (0.2s).
FINE: Initializing barback finished (0.9s).
FINE: Loading asset environment finished (1.3s).
ERR : The null object does not have a getter 'pubspec'.
    | 
    | NoSuchMethodError: method not found: 'pubspec'
    | Receiver: null
    | Arguments: []
FINE: Exception type: NoSuchMethodError
ERR : dart:core                                                                                            Object.noSuchMethod
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 251   _PackageDependencyComputer._PackageDependencyComputer
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 107   DependencyComputer._transformersNeededByTransformer
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 258   _PackageDependencyComputer._PackageDependencyComputer
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 199   DependencyComputer._loadPackageComputer
    | dart:core                                                                                            List.forEach
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/dependency_computer.dart 53    DependencyComputer.DependencyComputer
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart 33  loadAllTransformers.<async>
    | dart:isolate                                                                                         _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                            _Completer.completeError
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart 127  loadAllTransformers.<async>
    | dart:isolate                                                                                          _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                        Future.Future.microtask
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/load_all_transformers.dart  loadAllTransformers
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 512  AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>.<async>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart      AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                        progress
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 511  AssetEnvironment._load.<fn>.<async>.<fn>.<async>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                        Future.Future.sync
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 671  AssetEnvironment._withStreamErrors
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 510  AssetEnvironment._load.<fn>.<async>
    | dart:async                                                                                        _SyncCompleter.complete
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 672  AssetEnvironment._withStreamErrors.<fn>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                        _Future.then
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 488  AssetEnvironment._load.<fn>.<async>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
INFO     2015-06-24 00:13:31,150 containers.py:280] 
    | ===== asynchronous gap ===========================
    | dart:async                                                                                        _Future.then
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 477  AssetEnvironment._load.<fn>.<async>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                        Future.Future.microtask
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart      AssetEnvironment._load.<fn>
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                        progress
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 459  AssetEnvironment._load
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 82   AssetEnvironment.create.<fn>.<async>
    | dart:isolate                                                                                      _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                       _Future.then
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 74  AssetEnvironment.create.<fn>.<async>
    | dart:isolate                                                                                     _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                                       Future.Future.microtask
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart     AssetEnvironment.create.<fn>
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/log.dart 378                       progress
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/barback/asset_environment.dart 73  AssetEnvironment.create
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/build.dart 56              BuildCommand.onRunTransformerCommand.<async>
    | dart:isolate                                                                                     _RawReceivePortImpl._handleMessage
    | ===== asynchronous gap ===========================
    | dart:async                                                                             Future.Future.microtask
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/build.dart       BuildCommand.onRunTransformerCommand
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command/barback.dart 58  BarbackCommand.run
    | package:args/command_runner.dart 179                                                   CommandRunner.runCommand.<fn>
    | dart:async                                                                             Future.Future.sync
    | package:args/command_runner.dart 132                                                   CommandRunner.runCommand
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command_runner.dart 122  PubCommandRunner.runCommand.<async>.<fn>
    | dart:async                                                                             Future.Future.sync
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/utils.dart 107           captureErrors.<fn>
    | package:stack_trace/src/chain.dart 78                                                  Chain.capture
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/utils.dart 121           captureErrors
    | /tmp/tmp_ldPtI/dart-1.10.0/dart/sdk/lib/_internal/pub/lib/src/command_runner.dart 122  PubCommandRunner.runCommand.<async>
    | dart:isolate                                                                           _RawReceivePortImpl._handleMessage
---- End log transcript ----

ERROR    2015-06-24 00:13:31,526 containers.py:283] The command '/bin/sh -c pub --trace build' returned a non-zero code: 1
INFO     2015-06-24 00:13:31,526 containers.py:292] --------------------------------------------------------
ERROR    2015-06-24 00:13:31,526 instance.py:280] Docker build aborted: The command '/bin/sh -c pub --trace build' returned a non-zero code: 1
INFO     2015-06-24 00:13:31,526 health_check_service.py:101] Health checks starting for instance 0.


```

**Comment by nex3** [comment]
I believe this is a duplicate of #1183. It's been fixed, but the fix won't be released until 1.11.

**Comment by Andersmholmgren** [comment]
I can confirm that this issue is fixed if I run w the latest 1.11 dev build in the docker registry
